### ISSUE TITLE
Formatting improvements to A11y statement

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1597]
As a user of the A11y statement I need to be able to easily find the specific information in the statment that I need so that I can address my concerns quickly.

WHY are we building?
We wanted collapsible sections as readers might only be interested in specific sections that impact their personal situation. We were under a time crunch to get the page up for a presentation, and it’s  not ideal to use “Learn more” for so many links. 

WHAT are we building?
(1) Adding "skip to" links at the top of the statement. We think there should be 2:
- Skip to Known issues
- Skip to Contact form
(2) Remove all the collapsible/expandable "Learn more" links, but keep the collapsible sections that use descriptive links ( Our tests include, With assistive technology enabled, Not enough context, Compatibility with various assistive technologies, Screen readers only, Keyboard navigation )

VALUE created by our solution
An A11y statement with only descriptive links that still allows users to quickly get to issues that impact them as well as the contact us form. 

## Documentation and Artifacts
[Slack thread](https://uxcontent.com/learn-more-is-not-enough-a-case-for-descriptive-cta/?mc_cid=c21df6e4ae)
[Learn more is not enough](https://uxcontent.com/learn-more-is-not-enough-a-case-for-descriptive-cta/?mc_cid=c21df6e4ae) 

## Acceptance Criteria
When a user is looking for information in the A11y statment, they can find it easily and quickly.

* A11y
* Bilingualism
* Privacy considerations
* Security controls in place
* Measuring success and metrics

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (find stuff yourself)

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by YedidaZalik** [comment]
Asked Phil and he explained "The skip to main content is a special type of “skip link” to skip over repetitive areas of a page like the header navigation. In our case, the content is not repetitive, so I would make the skip links visible to all. Ours acts almost as a table of content (but not quite)"

Scheduled time for the skip links and then Yedida can finish rest of formatting

**Comment by YedidaZalik** [comment]
To make the sentence more readable in the skip link, Phil adjusted the header "Faites-nous part des problèmes d’accessibilité de Notification GC" to "Passer à Nous faire part des problèmes d’accessibilité de Notification GC"
Melissa reviewed and card can be moved to done

**Comment by amazingphilippe** [comment]
QA'd by Melissa and moving to done.
